src/styles/globalStyles.scss

```diff
@@ -34,8 +34,12 @@ body {
 
 img {
 	-webkit-user-drag: none;
-	-webkit-user-select: none;
+	user-select: none;
```

Comment:
why none?

Reply:
To prevent highlighting on dragging oar CTRL+A